### ISSUE TITLE
The paper link

### ISSUE THREAD

**Issue by Zplusdragon** [ChanglongJiangGit/A2J-Transformer#1]
very appreciate your contributions to this community. However, I can't find the paper link. When will it be released? thanks:)

**Comment by ChanglongJiangGit** [comment]
Of course, the link to the article has been updated now, thank you for your suggestion!
